source/api/filter.md

```diff
@@ -57,6 +57,28 @@ hexo.execFilterSync(type, data, options);
 hexo.extend.filter.unregister(type, filter);
 ```
 
+**Example**
+
+``` js
+// Unregister a filter which is registered with named function
+
+const filter = (data) => {
```

Comment:
suggest to named it something else (`imgFn` perhaps?) other than filter, otherwise may confuse with `hexo.extend.`**filter**

Reply:
Maybe I will rename it to filterFn.

Reply:
I initially thought of that as well, but there's still a `filter` in it. Ultimately, I think it should be fine, as long it's not `filter`.

Reply:
Updated.